### PR TITLE
Adds a handler add_tta_match_result

### PR DESCRIPTION
**Pull request by Dawsen**
1. This is currently a no-op. We do not have anything linking to this handler.
2. The add_tta_match_result page is identical to the
add_ffa_match_result page for now, but we will be working on
TTA-specific logic on this page.

### CODE REVIEW

File: src/static/add_tta_match_result.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width">
   <script src="/static/js/http.js" async=true></script>
-  <script src="/static/js/add_ffa_match_result.js" async=true></script>
+  <script src="/static/js/add_tta_match_result.js" async=true></script>
```
Comment (pg30123) [inline src/static/add_tta_match_result.html]: create add_tta_match_result.js as an empty file in the same pull request?

Comment (Dawsen) [reply src/static/add_tta_match_result.html]: I plan to start with a functional /add_tta_match_result, and then incrementally modify the code to support TTA-specific logic. I tried and when add_tta_match_result.js is empty, /add_tta_match_result is not functional.

If you think it's better to build from scratch for this handler, then I can make add_tta_match_result.js an empty file. What do you think?

Comment (pg30123) [reply src/static/add_tta_match_result.html]: Make sense, then maybe copy the add_ffa_match_result.js to add_tta_match_result.js?

Comment (Dawsen) [reply src/static/add_tta_match_result.html]: done

**Comment by pg30123** [comment]
LGTM ah

**Pull request merged**